stdlib/effect.mli

```diff
@@ -16,13 +16,21 @@
 
     @since 5.0 *)
 
-type _ t = ..
+type 'a t = 'a eff = ..
 (** The type of effects. *)
 
+exception Unhandled_effect : 'a eff -> exn
+(** Exception raised when an effect is performed and there is no matching
+    handler for that effect. The argument is the unhandled effect. *)
```

Comment:
This could be improved by doing the usual trick of writing an expression for the item and describe what it means/how it should be read. In this case:
```
(** [Unhandled e] is raised when effect [e] is performed and there is no handler for it. *)
```